src/Collection/index.js

```diff
@@ -153,6 +156,17 @@ export default class Collection<Record: Model> {
     })
   }
 
+  _subscribers: Array<(CollectionChangeSet<Record>) => void> = []
+
+  experimentalSubscribe(subscriber: (CollectionChangeSet<Record>) => void): () => void {
+    this._subscribers.push(subscriber)
+
+    return () => {
+      const idx = this._subscribers.indexOf(subscriber)
+      this._subscribers.splice(idx, 1)
```

Comment:
Shouldn't we check if `idx !== -1`? You can call this function many times

Reply:
shit, you're right here. I thought I checked that splicing with `-1` as first arg is a no-op, but… it's not. will fix.

Reply:
fixed for all subscribers